ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1526,6 +1526,40 @@ def test_sosreport_without_healthcheck_installed(self, remove_healthcheck):
             ]
         )
 
+    @pytest.fixture
+    def change_pwd_plugin_default(self):
+        """
+        Fixture to change the password plugin feature
+        to AllowNThash and change it to default
+        """
+        self.master.run_command(
+            ["ipa", "config-mod", "--ipaconfigstring=AllowNThash"]
+        )
+        yield
+        self.master.run_command(
+            ["ipa", "config-mod",
+             "--ipaconfigstring={AllowNThash, 'KDC:Disable Last Success'}"]
```

Comment:
The syntax `{AllowNThash, 'KDC:Disable Last Success'}` relies on bash brace expansion that is not available if you call run_command with an arg list.

In order to have a minimal change, you can use instead:
```
        self.master.run_command(
            ["ipa", "config-mod", "--delattr", "ipaconfigstring=KDC:Disable Last Success"]
        )
        yield
        self.master.run_command(
            ["ipa", "config-mod", "--addattr",
             "ipaconfigstring=KDC:Disable Last Success"]
```